.gitignore

```diff
@@ -1,6 +1,9 @@
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 
+# ignore local preferences
+*.userprefs
```

Comment:
If you use monodevelop, you should add the ignores of that editor to your global gitignore:
https://github.com/github/gitignore/blob/master/Global/MonoDevelop.gitignore